### PR TITLE
cleaning up repo 

### PR DESCRIPTION
**Pull request by wiederm**
## Description
Cleaning up the test data from the repo

## Todos
  - [x] remove data
 - [x] provide interface to access data from remote repo

## Status
- [x] Ready to go

### CODE REVIEW

File: transformato/tests/test_restraints.py [hunk 1]
```diff
@@ -25,12 +24,14 @@
 from simtk.openmm import *
 from simtk.openmm.app import *
```
[the remark this answers is not in the document]
Comment (JohannesKarwou) [reply transformato/tests/test_restraints.py:25]: Sure, you're right

File: transformato/tests/test_restraints.py [hunk 2]
```diff
@@ -10,136 +10,129 @@
...
+            + glob.glob(f"{PATH_2OJ9_DIR}../toppar/*.prm")
+        )
+    )
     psf = CharmmPsfFile(f"{PATH_2OJ9_DIR}step3_input.psf")
     crd = read_crd(f"{PATH_2OJ9_DIR}step3_input.crd")
 
-    top=gen_box(psf,crd)
-    
+    top = gen_box(psf, crd)
+
```
Comment (wiederm) [inline transformato/tests/test_restraints.py:135]: all of these functions/CLasses can be directly imported or defined as functions in the local scope. 
I think that would be preferable to importing functions from files.

File: transformato/tests/test_charmm.py [hunk 3]
```diff
@@ -22,11 +24,10 @@ def test_run_28_1h1q_rbfe_production_with_CHARMM():
     )
```
Comment (wiederm) [inline transformato/tests/test_charmm.py:25]: @JohannesKarwou maybe we can come up with something better here? (1) I am not sure if we should really ship these tests, and (ii) we should really think about how to provide a good CHARMM experience. Maybe related: is there a conda binary for CHARMM?

Comment (JohannesKarwou) [reply transformato/tests/test_charmm.py:25]: I totally agree with you, I'm also not that happy with these tests, but for now I have no better idea, but I sometimes need some tests, so currently I'm using them locally for me...
I don't know about a conda binary for CHARMM but I will ask Stefan. I just know that there is some pyCHARMM devolpment going on, maybe we can integrate that...

**Review by JohannesKarwou — APPROVED** [review]
I tested it locally and everything runs fine. This will reduce our storage drastically!

**Pull request merged**